### PR TITLE
SalesforceReact was not compiling in xcode/command line

### PR DESCRIPTION
**Pull request by wmathurin**
Now it is, and it also works from generated app (cocoapod)

### CODE REVIEW

**Comment by kchitalia** [comment]
:+1:

File: build/sdk.sh [hunk 1]
```diff
@@ -34,7 +34,6 @@ SCHEMES_TO_BUILD = [
   "NoteSync",
   "OCMock",
   "OCMockLib",
-  "React",
```
Comment (bhariharan) [inline build/sdk.sh]: Don't we need ```SalesforceReact``` here?

Comment (wmathurin) [inline build/sdk.sh]: It is there line 42.
The react scheme doesn't exist. (We don't compile the React project from the SDK workspace).

**Comment by bhariharan** [comment]
:+1:

**Pull request merged**